sunpy/sun/models.py

```diff
@@ -9,9 +9,9 @@
 
 Object
 ------
-    interior : pandas.DataFrame
+    interior : astropy.table.QTable
```

Comment:
Add a backtick on each side please.

Reply:
Okay, will do. Also there are a few additional changes that I need to make in the documentation part, which I'll do and make a commit.

Reply:
Done